docs/site/angular-auth-oidc-client/docusaurus.config.js

```diff
@@ -81,7 +84,13 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
-          editUrl: 'https://github.com/damienbod/angular-auth-oidc-client',
+          editUrl:
+            'https://github.com/damienbod/angular-auth-oidc-client/edit/main/docs/site/angular-auth-oidc-client',
+          remarkPlugins: [
+            [require('@docusaurus/remark-plugin-npm2yarn'), { sync: true }],
+          ],
+          showLastUpdateAuthor: true,
+          showLastUpdateTime: true,
```

Comment:
Because I find it useful, I can also revert this if you want

Reply:
Perfect, thanks!